src/Playlist.tsx

```diff
@@ -33,10 +37,21 @@ export const PlayList: React.FC = () => {
     return <div>Playlist Loading from {endpoint}</div>;
   }
 
+  if (itemClicked !== -1) {
+    return <Songlist playlistId={data.items[itemClicked].id}></Songlist>;
+  }
+  const onClick = (options: { key: number }): void => {
+    const { key } = options;
+    setItemClicked(key);
+  };
+
   const listPlaylists = (list: fetchData) => {
     return list.items.map((e, i) => {
```

Comment:
its usually a good idea to name your variables even in an implicit lambda function like this. I can tell this is `element` and `index` but explicitly naming this will make it easier for you to come back and read this later on